### ISSUE TITLE
Issue while predicting a new chest XRay image

### ISSUE THREAD

**Issue by prasanth-rgb** [zoogzog/chexnet#22]
Hi,

I got a strange behavior when tried to predict a new image with the fresh code from github.

At the end of prediction, it got very less rate 2.34 e-2 (near values) in outPRED for all the classes and when i take the F.softmax(outPRED,dim = 0), it give a tensor with value [1. ,1. ,1. ,1. ,1. ,1. , 1. , 1. ,1. ,1. , 1. , 1. ,  1. , 1.]

Any idea on why it is getting this way?

**Comment by agil27** [comment]
I haven't gotten into the details of this piece of code, but as far as I am concerned about the problem setting here, the Chest X-ray pathology prediciton is a **multi-label** problem rather than a **multi-class** problem, which means the output logits won't necessarily add up to 1, and thus softmax, as a normalization function, shouldn't be used here.

**Comment by stupidBirdFirstFly** [comment]
Hi,I am new to deep learning.Can you run this code? If it can run, I would like to ask about your operating environment, including pytorch version, graphics card model, cuda, etc.Thank you very much.
